src/components/Document.js

```diff
@@ -460,31 +459,57 @@ class DocumentPreviewSection extends Component {
         if (!type || !url) return null;
 
         let preview = null;
-        if (type == "application/pdf") {
-            const pdfViewerUrl = `/viewer/web/viewer.html?file=${encodeURIComponent(url)}`;
-            preview = ( <>
-                          <p> Annotate this document in the <a target="_blank" href={pdfViewerUrl}>PDF viewer</a>. </p>
-                          <div id="hoover-pdf-viewer-container" className={classes.preview}>
-                            <iframe
-                                src={pdfViewerUrl}
-                                height="100%"
-                                width="100%"
-                                allowfullscreen="true"
-                            />
-                          </div>
-                        </> );
-        } else if (type.startsWith("image/") || type.startsWith("audio/") || type.startsWith("video/")) {
-            preview = ( <div id="hoover-media-viewer-container" className={classes.preview}>
-                        <embed
-                            style={{"object-fit": "contain"}}
-                            src={url}
-                            type={type}
+        if (type == 'application/pdf') {
+            const pdfViewerUrl = `/viewer/web/viewer.html?file=${encodeURIComponent(
+                url
+            )}`;
+            preview = (
+                <>
+                    <p>
+                        {' '}
+                        Annotate this document in the{' '}
+                        <a target="_blank" href={pdfViewerUrl}>
+                            PDF viewer
+                        </a>
+                        .{' '}
+                    </p>
+                    <div
+                        id="hoover-pdf-viewer-container"
+                        className={classes.preview}>
+                        <iframe
+                            src={pdfViewerUrl}
                             height="100%"
                             width="100%"
-                            title={docTitle}
+                            allowfullscreen="true"
                         />
-                        </div> );
-
+                    </div>
+                </>
+            );
+        } else if (
+            type.endsWith('/pjpeg') ||
```

Comment:
These are many enough that they deserve their own list.

I see you're checking the ends to catch any of [`application/mp4`, `video/mp4`, `audio/mp4`](https://tools.ietf.org/html/rfc4337) and so on.

I wonder where this list comes from. I should probably look for a list of mime types that Firefox can `<embed>` by default.